.github/SECURITY.md

```diff
@@ -6,9 +6,11 @@ We support fixing security issues on the following releases:
 
 | Version | Supported          | Security fixes until
 | ------- | ------------------ | -------------------- 
-| 4.4     | :white_check_mark: | 24 Months after the release of 5.0
-| 4.3     | :white_check_mark: | Release of 5.0
-| 4.2     | :white_check_mark: | Release of 4.5
+| 5.0     | :white_check_mark: | 24 Months after the release of 5.0
+| 4.5     | :white_check_mark: | 24 Months after the release of 5.0
+| 4.4     | :x:                | 24 Months after the release of 5.0
```

Comment:
This should be "No longer supported" I think

Reply:
I'm fine doing security releases on 4.4 for a while longer. We don't have many usually.

Reply:
Yea, you will do a one-off for a real issue, but not sure how we state that. Leave it to you.